test/test-repl-interaction.js

```
@@ -0,0 +1,248 @@
+    
+    // Terminate the REPL process
+    const terminated = terminalManager.forceTerminate(result.pid);
+    if (!terminated) {
+      console.warn(`${colors.yellow}Warning: Could not terminate Python REPL process${colors.reset}`);
+    } else {
+      console.log(`${colors.green}✓ Terminated Python REPL process${colors.reset}`);
+    }
+    
+    // Check if we got the expected output
+    if (output && output.includes(`REPL_TEST_VALUE: ${testValue * 2}`)) {
+      console.log(`${colors.green}✓ Python REPL test passed!${colors.reset}`);
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)